src/main/java/com/crystalcraftmc/crystalstrike/BuildCrystal.java

```diff
@@ -0,0 +1,190 @@
+package com.crystalcraftmc.crystalstrike;
+
+//import org.bukkit.plugin.PluginManager;
+//import org.bukkit.plugin.java.JavaPlugin;
+//import org.bukkit.ChatColor;
+//import org.bukkit.event.EventHandler;
+//import org.bukkit.event.Listener;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.Chest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bukkit.Location;
+import org.bukkit.event.weather.LightningStrikeEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+
+
+
+public class BuildCrystal {
+	
+	private Location baseLocation;
+	private final LightningStrikeEvent event;
+	
+	public BuildCrystal(final LightningStrikeEvent e){
+		baseLocation = e.getLightning().getLocation();
+		event = e;
+	}
+	
+	public void firstCrystal(){
```

Comment:
JavaDoc method comments would be helpful here too. I'm following what's happening but best to comment.